packages/transit-vehicle-overlay/src/utils.js

```diff
@@ -0,0 +1,298 @@
+import polyline from "@mapbox/polyline";
+
+/** deep copy */
+export function deep(obj) {
+  return JSON.parse(JSON.stringify(obj));
```

Comment:
This has [a high potential to result in data loss](https://stackoverflow.com/a/122704/269834). Please use lodash.clonedeep.